smarts/core/tests/test_scenario.py

```diff
@@ -98,6 +102,6 @@ def test_scenario_variations_of_social_agents(scenario_root):
         for group, speed in itertools.product(groups, speeds)
     }
 
-    assert (
-        len(all_social_agent_ids - expected_social_agent_ids) == 0
+    assert (len(all_social_agent_ids & expected_social_agent_ids)) == len(
+        expected_social_agent_ids
```

Comment:
I made this length check explicit since the old check was subtracting a superset from the resulting set.